### PR TITLE
NMS-10376: Document RPC TTL tuning

### PR DESCRIPTION
**Pull request by j-white**
JIRA: https://issues.opennms.org/browse/NMS-10376

### CODE REVIEW

File: opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc [hunk 1]
```diff
@@ -5,20 +5,61 @@
...
+For cases where we cannot derive a suitable TTL, a global value is used wich defaults to `20000` (20 seconds).
+This value can be configured.
 When using the JMS-based RPC implementation, set value of the `org.opennms.jms.timeout` system property to the desired number of milliseconds.
 When using the Kafka-based RPC implementation, set the value of the `org.opennms.core.ipc.rpc.kafka.ttl` system property to the desired number of milliseconds.
 
+==== Using meta-data for TTLs
+
+For RPCs that derive TTLs from service parameters, the link:#ga-meta-data[meta-data] feature can be used to customize these values on a node/interface/service basis.
+
+For example, the ICMP detector could be configured to use the value of the `ttl` associated with the node meta-data, or default to 30 seconds if none is setup as follows:
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc]: set up vs. setup?

File: opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc [hunk 2]
```diff
@@ -5,20 +5,61 @@
...
+2. Fall-back to using the service interval as the TTL (this is rate at which the service is scheduled)
+
+These rules apply to all monitors and collectors, excluding the `SnmpCollector`.
+
+==== TTLs for the SNMP Collector
+
+The TTL used when invoking the `SnmpCollector` remotely is:
+
+1. The value of the `ttl` associate with the SNMP agent's configuration
+2. Fall-back to using the service interval as the TTL (this is rate at which the service is scheduled)
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc]: rate vs. the rate

File: opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc [hunk 3]
```diff
@@ -5,20 +5,61 @@
...
+
+==== TTLs for the other SNMP communication
+
+The TTL used when invoking other types of SNMP requests remotely is:
+
+1. The value of the `ttl` associate with the SNMP agent's configuration
+2. Fall-back to using the service interval as the TTL (this is rate at which the service is scheduled)
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc]: rate vs. the rate

File: opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc [hunk 4]
```diff
@@ -5,20 +5,61 @@
...
+TIP: To troubleshoot TTLs, consider enabling the link:#opentracing[OpenTracing] integration on both _{opennms-product-name}_ and _Minion_.
 
-If no TTL is found in any of the ways specified above, a default TTL of `20000` msec (20 sec) will be used.
-The default TTL can also be configured.
+==== TTLs in Pollerd & Collectd
+
+The TTL used when invoking monitors and collectors remotely is:
+
+1. The value from the `ttl` parameter associated with the service
+2. Fall-back to using the service interval as the TTL (this is rate at which the service is scheduled)
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc]: rate vs. the rate

File: opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc (hunk 1 above)
Comment (j-white) [reply opennms-doc/guide-admin/src/asciidoc/text/minion/ttl.adoc]: setup is correct in this context

**Comment by j-white** [comment]
Thanks for the review @indigo423.

**Pull request merged**